webapi/src/main/scala/org/knora/webapi/responders/v1/OntologyResponderV1.scala

```diff
@@ -707,11 +698,23 @@ class OntologyResponderV1(responderData: ResponderData) extends Responder(respon
   private def getPropertyTypesForResourceType(
     resourceClassIri: IRI,
     userProfile: UserADM
-  ): Future[PropertyTypesForResourceTypeResponseV1] =
+  ): Task[PropertyTypesForResourceTypeResponseV1] =
     for {
-      resInfo: ResourceTypeResponseV1 <- getResourceTypeResponseV1(resourceClassIri, userProfile)
-      propertyTypes                    = resInfo.restype_info.properties.toVector
-
+      resInfo      <- getResourceTypeResponseV1(resourceClassIri, userProfile)
+      propertyTypes = resInfo.restype_info.properties.toVector
     } yield PropertyTypesForResourceTypeResponseV1(properties = propertyTypes)
+}
 
+object OntologyResponderV1Live {
+  val layer: URLayer[
+    StringFormatter with TriplestoreService with MessageRelay with IriService with AppConfig,
```

Comment:
```suggestion
    StringFormatter with MessageRelay,
```
Same as in the other PR, the unused "dependencies" could probably be removed from the environment?

Reply:
Thanks.